src/axom/inlet/docs/sphinx/quick_start.rst

```diff
@@ -0,0 +1,124 @@
+.. _inlet_quick_start_label:
+
+Quick Start
+===========
+
+Inlet's workflow is broken into the four following steps:  
+
+* Defining the schema of your input deck
+* Verifying that the user given input deck is valid
```

Comment:
Suggestion for second step:  
> Read in and validate the user-provided input

Reply:
reading happens during the defining stage. It has been suggested to break reading and defining apart.